core/chains/evm/client/errors.go

```diff
@@ -184,13 +184,14 @@ var clients = []ClientErrors{parity, geth, arbitrum, optimism, metis, substrate,
 // https://github.com/klaytn/klaytn/blob/dev/blockchain/error.go
 // https://github.com/klaytn/klaytn/blob/dev/blockchain/tx_pool.go
 var klaytn = ClientErrors{
-	NonceTooLow:                 regexp.MustCompile(`(: |^)nonce too low$`),
-	TransactionAlreadyInMempool: regexp.MustCompile(`(: |^)(known transaction|there is another tx which has the same nonce in the tx pool$)`), // `known transaction` error could end with additional information
-	TerminallyUnderpriced:       regexp.MustCompile(`(: |^)(transaction underpriced|^intrinsic gas too low)`),
-	LimitReached:                regexp.MustCompile(`(: |^)txpool is full`),
-	InsufficientEth:             regexp.MustCompile(`(: |^)insufficient funds`),
-	TxFeeExceedsCap:             regexp.MustCompile(`(: |^)(invalid gas fee cap|max fee per gas higher than max priority fee per gas)`),
-	Fatal:                       gethFatal,
+	NonceTooLow:                       regexp.MustCompile(`(: |^)nonce too low$`),
+	TransactionAlreadyInMempool:       regexp.MustCompile(`(: |^)(known transaction)`), // `known transaction` error could end with additional information
+	ReplacementTransactionUnderpriced: regexp.MustCompile(`(: |^)replacement transaction underpriced$|there is another tx which has the same nonce in the tx pool$`),
+	TerminallyUnderpriced:             regexp.MustCompile(`(: |^)(transaction underpriced|^intrinsic gas too low)`),
+	LimitReached:                      regexp.MustCompile(`(: |^)txpool is full`),
```

Comment:
Can you provide more details about this error in Klaytn?
What do you recommend the client should do?
Retry the same tx after some time, or increase the gas?

Reply:
I added more comments for them. PTAL again.